### ISSUE TITLE
__content.json Did Not Render Properly

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#69]
# Description
```
Please provide a short description of what you have encountered below.
```
The `__content.json` is supposedly a working JSON. However, it is being rendered as a large pile of string. This is not seen if the data file is via `__data.toml` approach.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
The output should be a working JSON. Here is the workaround patch:

```
From 785df489335e4075dc8ca14918e1f1fa5e9c9ba7 Mon Sep 17 00:00:00 2001
From: "(Holloway) Chew, Kean Ho" <hollowaykeanho@gmail.com>
Date: Mon, 13 Feb 2023 13:54:14 +0800
Subject: [PATCH 1/2] sites: patched a workaround for __content.json to render
 properly.

The existing hestiaHUGO renders __content.json inappropriately where
it makes the entire thing a string. Hence, I need to workaround it
and upstream the patch. Hence, let's do this.

This patch workarounds a rendering bug caused by hestiaHUGO hugo
theme for rendering __content.json properly in sites/ directory.

Signed-off-by: (Holloway) Chew, Kean Ho <hollowaykeanho@gmail.com>
---
 .../data/hugo/process/Content/.Init.swp         | Bin 0 -> 12288 bytes
 .../functions/data/hugo/process/Content/Init    |  10 +++++++++-
 2 files changed, 9 insertions(+), 1 deletion(-)
 create mode 100644 sites/themes/hestiaHUGO/layouts/partials/Hestia/functions/data/hugo/process/Content/.Init.swp

diff --git a/sites/themes/hestiaHUGO/layouts/partials/Hestia/functions/data/hugo/process/Content/.Init.swp b/sites/themes/hestiaHUGO/layouts/partials/Hestia/functions/data/hugo/process/Content/.Init.swp
new file mode 100644
index 0000000000000000000000000000000000000000..2f989f095bc36175c851a14429ab9c6d6fa6a7f8
GIT binary patch
literal 12288
zcmeI2L2nyH6vwBf5-PM!1vgHQB}HkK>;}W3Dh;Ze7-E{*k)5<E7c^dvy`!vmwzIQw
zLsbe_#0P*6P&gwYJ_1*yN+2#EaRJIVKth85?5>^0P3gJnSo+uAoqccK`@J_?ReinX
zYt5^4-mfuyPBZq|4-djHs<B`06U&$F%_o08QkuuSm#Ds2ic}n{G2c7P>f?MlSW{s>
z5{b!z$&<mR(q^Dyse()zkp(6aBYZ_7Gm@`wzt;|8zNd1Sq+H`xoCWK}S}@3y&`6bJ
zL7y8QMEOt!saB!LvS3vu2KxlfM4H2barYd7XNf>=221BpvNvmQ6sP(%?^SyFrR`^l
zfXmepa0DCyN5Bzq1RMcJz!7-n37GT<y9)|OD-tW!_mP?J>e78V0*-(q;0QPZj({WJ
z2si?cfFs}tI0BBqQ%HdK8T<En#^#PAc>MqW<lp~)&oTBL^eOZSbQcPs-%c_16Z9kW
zCG-Wf4*heIv2UOO^dWQ!It|T1r=XM23F!U_*g*s6E$9{K81&bRj6Hz<g#LhjhrWmI
zK@wVk=AfTnVC-|K4PAy_ht5Fr&@t#a=qU6n^7$6}8u|*l2i=4O)Q3W-2XUwY75P7f
z^>ZOR0*-(q;0QPZj==5Px0gr@lk24TwSP$?p@)LzH#z=X=*!TMhY>*-9I)8GdW?ac
zDfDRG&KqvX%$%jAJ9n713eI8Uq%SmKR58sBoePRJTNJyFwc`4?y_nk5%4DC3$11q_
z9hyfXrGLf>tEr392bN)OG*7aD(j$L68F8IOJT5HvH<JV6*xb6K1#y$$+Izp-Zg~fs
z<L(OOX^I3EGNWZO#DfQ)C^xiz{C1h6P5zM3syR*a9+}e&F|4U#5j3<`dH|5sjWXIY
zugR?`nwgqe2i~NHS262}lJ0vtTgH5R)>n#@%%CW`t>FbavkN1e3);ZCJOJ$Qg#zIU
zJwCl+47H?J44#%d?0c>p9MX%;1U)m&Y7OlL+ubUeqAPh~`*&v%x9P$P*|2RQh4ZQb
ztJzobJ4Gjo{%jLYal^p?gS2JRMMv7=K6-cBl~^hc&7|GSgY>u(m8ZlSl+`G9D9ZT%
z7DY_6CfNE3#8+d%6Oa6ZECj3sjg{W9&AiLC<h@vAgu_B6lK4@L56@D<QTA~dig+ym
zk_nZaOkJb%5U&?8v8l*6TbtWkv{~=euQs+Co$h40dI@By2<1SA6r+l{eFY8GuF#1M
z3@2_Rv&@b$7w0sVBWZXs)=YWvLcu^ZZOBk0nef?ayS3KbYPMTlYQIbM)(yJSY^^Pl
zka$d!xRq)g2||u?l*Ur@ed-E<gr`0#0D?W@odBhex0;r{+@O0@f{4hgcd6O+=yJW=
z>@L#v=GJ-}$zHE_I`!68vw=W5Y&!wk46neOxzsT2?GaD0e`uK_?XiuB54FIWQI_af
zON0CuDN_S8wxxP#YsP5y6B(0U2!io=?DLd|k?@rs2H3%J$%2jMYNOR{EY*C>w^p=>
z8&+0YA_#FS6sfUWhsvViNr?6Uwuj;PbVKDd;(MfehT~a|r&C-F>{AoR{f8=3FM^pA
zXpyc7ZHq|=gcdAzuZqSy`!->Yy>Ol=LwP3fcf%^YYdMSz)s|~D`k>vZZ}1*9aJV(z
zSvaB>jR>Pl)sVf9*x#8o^46=VdyiHlF<zuAI1jY0=;983_>uB~Ykbf}*|_XKLr=41

literal 0
HcmV?d00001

diff --git a/sites/themes/hestiaHUGO/layouts/partials/Hestia/functions/data/hugo/process/Content/Init b/sites/themes/hestiaHUGO/layouts/partials/Hestia/functions/data/hugo/process/Content/Init
index 293c3fe..7fa7dbc 100644
--- a/sites/themes/hestiaHUGO/layouts/partials/Hestia/functions/data/hugo/process/Content/Init
+++ b/sites/themes/hestiaHUGO/layouts/partials/Hestia/functions/data/hugo/process/Content/Init
@@ -43,7 +43,15 @@ specific language governing permissions and limitations under the License.
 	{{- $ret = $ret.Output -}}
 
 
-	{{- $dataList = merge $dataList (dict (upper (string $k)) $ret) -}}
+	{{- $k = upper (string $k) -}}
+
+
+	{{- if and $ret (eq $k "JSON") -}}
+		{{- $ret = transform.Unmarshal $ret -}}
+	{{- end -}}
+
+
+	{{- $dataList = merge $dataList (dict $k $ret) -}}
 {{- end -}}
 
 
-- 
2.39.1
```

# Current Behavior
```
Please specify the current behavior of software below.
```
rendered JSON is a large string.

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
fixed in 5affb38dc99c779afe6bf923a074030be70d6cc1

**Comment by hollowaykeanho** [comment]
Released in `v1.1.0`.
